sklearn/linear_model/passive_aggressive.py

```diff
@@ -183,10 +195,14 @@ class PassiveAggressiveRegressor(BaseSGDRegressor):
         Whether the intercept should be estimated or not. If False, the
         data is assumed to be already centered. Defaults to True.
 
-    n_iter : int, optional
-        The number of passes over the training data (aka epochs).
+    max_iter : int, optional
+        The maximum number of passes over the training data (aka epochs).
+        The maximum number of iterations is set to 1 if using partial_fit.
```

Comment:
I would rather say that this parameter only impacts the behavior of the `fit` method, not the `partial_fit` method.